setup.py

```diff
@@ -304,6 +306,7 @@ def do_setup():
             'sendgrid' : sendgrid,
             'slack': slack,
             'ssh': ssh,
+            'winrm': winrm,
```

Comment:
Add your entry at the correct place to ensure alphabetic ordering.

Reply:
fixed, commit coming